app/models/katello/sync_plan.rb

```diff
@@ -31,7 +31,7 @@ class SyncPlan < Katello::Model
     scoped_search :on => :enabled, :complete_value => true
 
     def validate_sync_date
-      errors.add :base, _("Start Date and Time can't be blank") if self.sync_date.nil?
+      fail HttpErrors::BadRequest, _("Sync Date format is incorrect.") if self.sync_date.nil?
     end
```

Comment:
You might just get rid of this method altogether and replace it with:
```
    validates :sync_date, :presence => true
```

towards the top of the model.  I'm not sure why this even exists :)